Makefile

```diff
@@ -46,13 +46,18 @@ DOCKER_RUN_CMD  = docker run ${DOCKER_FLAGS} ${BUILD_TOOLS_DOCKER_IMAGE}
 .PHONY: default
 default: help
 
-## Run pre-commit hooks in build-tools docker container.
+# not exported in make help on purpose, as this is a one-time shot to make life easier
+.PHONY: template/adjust
```

Comment:
tbH, after seeing this a couple of times and thinking about it. This should rather be a separate cool or simple alias in your local bashrc rather than part of the repo

Reply:
actually this should be a pre-commit to verify template cleanup has been done... like e.g. terraform fmt...

Reply:
okay, so what are the next steps here? Shall we put this to our backlog?

Reply:
not a priority at all.. let's see how often this process fails...